website/content/docs/secrets/aws.mdx

```diff
@@ -474,6 +474,38 @@ have a time-to-live of 1 hour and automatically refresh when they expire.
 Please see the [API documentation](/vault/api-docs/secret/aws#configure-root-credentials)
 for more details on the fields associated with plugin WIF.
 
+## Cross-Account Management of Static Roles
+
+<EnterpriseAlert product="vault" />
+
+Vault now supports cross-account access for AWS static roles,
+allowing credential management for IAM users across multiple AWS accounts via AWS Security Token Service (STS).
```

Comment:
```suggestion
Vault supports cross-account access for AWS static roles. You can configure credential management for IAM users across multiple AWS accounts by assuming roles in the other accounts.
```

Suggest to use active voice, as per Education team recommendations. I also made some tweaks to make it more of an elevator pitch. What do you think?